include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -8874,6 +8874,15 @@ def err_bounds_type_annotation_lost_checking : Error<
   def err_expected_bounds : Error<
     "expression has no bounds">;
 
+  def err_expected_bounds_for_ptr_cast : Error<
+    "expression has no bounds, cast to ptr<T> require a source /w bounds">; 
```

Comment:
There's a typo in the error messages to correct: "require" should be "requires".    

Please don't abbreviate in the error messages: decl should be expanded to declaration the word "with" should be expanded and so on.